### ISSUE TITLE
Autoscroll not working when using Nesting

### ISSUE THREAD

**Issue by idoodler** [computerjazz/react-native-draggable-flatlist#425]
**Describe the bug**
When using `NestableScrollContainer` and `NestableDraggableFlatList` autoscroll is not working. We are using ReactNativeWeb. Also I edited the snack you provided to add the following props to the `NestableDraggableFlatList` 
```
autoscrollSpeed={5}
autoscrollThreshold={20}
```

**Platform & Dependencies**
Please list any  applicable dependencies in addition to those below (react-navigation etc).
 - react-native-draggable-flatlist version: 3.1.2
 - Platform: Web
 - React Native (Web) version: 0.17.7
 - Reanimated version: 2.8.0
 - React Native Gesture Handler version: 2.4.2

**Comment by TomCorvus** [comment]
Hi there,
Same problem with `react-native-draggable-flatlist: 4.0.0-beta.12`

**Comment by computerjazz** [comment]
yeah, this is a known web bug. web treats containers differently than native -- in RN, if I wrap a scrollable list in a container, the container does not expand beyond the boundaries of the screen, and so the list can calculate whether the drag is near the edge of the container and scroll accordingly. In web, the container expands to the height of the content and overflows offscreen if need be, it doesn't know that the drag is nearing the edge of the screen. It's a tricky one to solve.

**Comment by TomCorvus** [comment]
@computerjazz The problem is the same on iOS or Android app.
